tests/test_MatchPessimisticB.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
```

Comment:
No `#!` line on unit tests. Sorry, but it breaks on Macs and so has been forbidden by our masters